static/sass/styles.scss

```diff
@@ -409,3 +409,9 @@ code {
 .u-text-muted {
   color: $color-mid-dark;
 }
+
+.p-navigation {
+  .p-navigation__link.is-selected > a::before {
+    background-color: $ubuntu-orange;
```

Comment:
I think you could achieve this by updating the value of [`$color-accent`](https://github.com/canonical-web-and-design/snapcraft.io/blob/master/static/sass/styles.scss#L4) to be `$ubuntu-orange`. This is the variable used by default in the navigation pattern, and the only other pattern it affects is the `is-ticked` list icon. Snapcraft does use that pattern on the [docs search](https://snapcraft-io-3592.demos.haus/docs/search?q=ooooooo), but if we could live with those being Ubuntu orange it would mean less overrides.

Including these links to show what changing the `$color-accent` variable would impact:
https://github.com/canonical-web-and-design/vanilla-framework/search?q=color-accent
https://github.com/canonical-web-and-design/snapcraft.io/search?q=color-accent

Reply:
@long-chung @AnnSoMuller do either of you have any objections to the ticks on [the docs search](https://snapcraft-io-3592.demos.haus/docs/search?q=ooooooo) being Ubuntu orange?

Reply:
Just had a chat with @lyubomir-popov about this - it's a little weird that `$color-accent` ties the nav highlight and is-ticked list items like this, so we're gonna look at whether that's something we can address in Vanilla directly.